custom-script/machine-learning-evalscript.ipynb

```diff
@@ -561,11 +505,13 @@
     "merge_task = MergeFeatureTask(input_features=[BANDS_SUB_FEATURE, NDWI_FEATURE, NDMI_FEATURE], output_feature=FEATURES)\n",
     "\n",
     "# Task for pixels' sampling\n",
-    "sampling_task = PointSamplingTask(\n",
-    "    n_samples=N_SAMPLES,\n",
-    "    ref_mask_feature=LABELS[1],\n",
-    "    ref_labels=REF_LABELS,\n",
-    "    sample_features=[FEATURES, IS_DATA_FEATURE, LABELS],  # tag fields to sample\n",
+    "sampling_task = BlockSamplingTask(\n",
+    "    amount=100,\n",
```

Comment:
I assume this should be `N_SAMPLES` and not hardcoded to 100?

Reply:
done